posthog/temporal/data_imports/pipelines/postgres/postgres.py

```diff
@@ -84,6 +84,39 @@ def _get_primary_keys(cursor: psycopg.Cursor, schema: str, table_name: str) -> l
     return None
 
 
+def _get_table_chunk_size(cursor: psycopg.Cursor, schema: str, table_name: str, logger: FilteringBoundLogger) -> int:
+    try:
+        query = sql.SQL("""
+            SELECT SUM(pg_column_size(t.*)) / COUNT(t.*) FROM (
+                SELECT * FROM {}
+                LIMIT 100
```

Comment:
Isn't it kind of pointless to do `LIMIT 100` and divide by count?

`pg_column_size` returns the number of bytes used to store a data value. But this is given by the schema: E.g. calling this function on a `bigint` column will always return 8 bytes, regardless if the actual value stored is 0 or 9223372036854775807. 

So, we could just do:
```
            SELECT pg_column_size(t.*) FROM (
                SELECT * FROM {}
                LIMIT 1
            ) as t
```

Reply:
I tested it out, but the wildcard returns some strange values (I still think my original comment is correct, but trying to figure out a way to reflect this in the query that doesn't require knowing all column names before hand)

Reply:
So, doing this: 
```
            SELECT SUM(pg_column_size(t.*)) FROM (
                SELECT * FROM {}
                LIMIT 1
            ) as t
```
Would return the size to construct a record with data values given by all columns in table. Which is maybe what we want?

Reply:
Nah, this is correct I believe. the count(*) is using the value from the `LIMIT` to get the average per row. `pg_column_size` can change depending on the row, for example:

```
select sum(pg_column_size(t.*)) from (
    select * from billing_upcominginvoice
    where id = 6190332
) as t
```
Returns 37,560

Whereas
```
select sum(pg_column_size(t.*)) from (
    select * from billing_upcominginvoice
    where id = 6211137
) as t
```
Returns 37,512

We're getting an average of the row size for 100 rows basically